piety-promoted-v1/modernized/02-piety-promoted-v1.adoc

```diff
@@ -8460,6 +8477,10 @@ so was his latter end comfortable and full of peace; in which he departed this l
 the 1st of the Tenth month, 1705, aged fifty-one years;
 and was buried at Friends`' burying-ground, at Sedbergh meeting-house.
 
+[.embedded-content-document.testimony]
+--
+
+[.postscript]
```

Comment:
yeah , I would mark this as a letter-heading, and then put a signature tag above his name below (Isaac Alexander). Also it will need a -- after the embedded section.